docs/features/dry-run.md

```
@@ -0,0 +1,59 @@
+oc-mirror -c ./isc.yaml --dry-run --workspace file:///home/user/oc-mirror/workspace docker://registry.example.com
+```
+
+## Output files
+
+Dry-run output is written to `working-dir/dry-run/` under the workspace or destination directory.
+
+### mapping.txt
+
+Lists every image that would be mirrored, in `source=destination` format:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)